src/node.cc

```diff
@@ -1392,7 +1391,7 @@ void FatalException(Isolate* isolate,
       // The fatal exception function threw, so we must exit
       ReportException(env, fatal_try_catch);
       exit(7);
-    } else if (caught->IsFalse()) {
+    } else if (caught.ToLocalChecked()->IsFalse()) {
```

Comment:
@addaleax do you know how `MaybeLocal`s work in a `TryCatch` scope? Is it OK to assume that when `caught` is empty, then `fatal_try_catch.HasCaught()` is also true?

Reply:
> Is it OK to assume that when `caught` is empty, then `fatal_try_catch.HasCaught()` is also true?

Yes, they should work the same way :)